utils/appliance.py

```diff
@@ -1385,6 +1413,137 @@ def find_by_name(self, appliance_name):
         return None
 
 
+def sprout_provision_appliance(vm_name=None, browser_steal=False, **kwargs):
```

Comment:
Thinking out loud a bit:
All of the `spout_*` methods look generally useful, but I'm thinking this is probably the beginning of some sprout consolidation that we need to do anyway. What do you think about putting these into `utils.sprout`? I think it might make sense for `SproutAppliance` to go in there, too.